src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,694 @@
+
+## Overview [#overview]
+
+Before using this tutorial, we first recommend reading these docs:
+
+* [Where to find user management features in the UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where)
+* [Important user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)
+* [SCIM API reference](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management/)
+* SCIM 2.0 RFC documents from the Internet Engineering Task Force: [RFC 7642](https://tools.ietf.org/html/rfc7642), [RFC 7643](https://tools.ietf.org/html/rfc7643), [RFC 7644](https://tools.ietf.org/html/rfc7644)
+
+This tutorial will walk you through how to:
+
+* [Configure your Authentication Domain for SCIM](#configure-scim)
+* [Create users and user groups in your system](#create-in-idp)
+* [Connect to the SCIM API](#connect)
+* [Create users in your Authentication Domain](#create-users)
+* [Create groups in your Authentication Domain](#create-groups)
+* [View users and groups in your Authentication Domain](#view)
+* [Update a user's attributes](#update-user)
+* [Update a group's members](#update-group)
+* [Delete users and groups](#delete)
+* [Manage user type with SCIM](#scim-user-type)
+
+## Configure your Authentication Domain for SCIM [#configure-scim]
+
+Before you can use the SCIM API, you must first [enable SCIM for your authentication domain](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#source-users). Note the API access token will be shown only once after saving the configuration, so keep it someplace safe! 
+
+<Callout variant="tip">
+If you need to view a bearer token later, the only way to do this is to generate a new one, and that will invalidate the old one and any integrations using the old token.
+</Callout>
+
+## Create users and user groups in your system [#create-in-idp]
+
+The SCIM API is typically used for importing users and groups into New Relic from a third-party identity provider. Create users and groups in your system before proceeding in this tutorial.
+
+If you want to use the SCIM API in a custom application or for ad-hoc requests, proceed to learn how to connect to the SCIM API.
+
+## Connect to the SCIM API [#connect]
+
+The SCIM API is available at `https://scim-provisioning.service.newrelic.com/scim/v2` and this URL is viewable in the Authentication Domain settings page. To access the SCIM API, your client must include a **bearer token** with each request. The token is displayed after saving your Authentication Domain configuration.
+
+If you are using a third-party identity provider, configure it to use **Bearer Token authorization** and plug in your API access token. Refer to your identity provider's documentation for help configuring this. Once configured, you're all set to import users and groups!
```

Comment:
There are messages like this in almost every paragraph. It feels like you might want a <Callout> that links to a separate tutorial or a section at the top that really explains, "seriously, if you're using a third-party idp, you probably shouldn't be in this document".